build/patches/harfbuzz/0004-disable-test-build.patch

```diff
@@ -0,0 +1,60 @@
+diff --git a/src/Makefile.am b/src/Makefile.am
+index b63bbe117..4345f0436 100644
+--- a/src/Makefile.am
++++ b/src/Makefile.am
+@@ -324,54 +324,9 @@ harfbuzz.cc: Makefile.sources
```

Comment:
Why not just run `make install-libLTLIBRARIES install-pkgconfigDATA` in the `src` dir?

Reply:
Interesting, I didn't knew about it, i will try it out